pkg/kubelet/dockershim/network/kubenet/kubenet_linux.go

```diff
@@ -844,6 +841,29 @@ func (plugin *kubenetNetworkPlugin) getRangesConfig() string {
 	return strings.Join(ranges[:], ",")
 }
 
+// given a n cidrs assigned to nodes,
+// create bridge routes configuration that conforms to them
+func (plugin *kubenetNetworkPlugin) getRoutesConfig() string {
+	var (
+		routes       []string
+		hasV4, hasV6 bool
+	)
+	for _, thisCIDR := range plugin.podCIDRs {
```

Comment:
is plugin.podCIDRs available if dual-stack is disabled? should we fall back to podCIDR?

Reply:
it is available either ways..